packages/js/src/browser.ts

```diff
@@ -88,7 +88,10 @@ class Honeybadger extends Client {
 
   public factory(opts?: Partial<Types.BrowserConfig>): this {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    return new Honeybadger(opts) as any
+    const clone = new Honeybadger(opts) as any
+    clone.setNotifier(this.getNotifier())
```

Comment:
This is important. I could go for a _static_ `__notifier` that is shared for all instances of the client, but I chose this method for more flexibility.

Reply:
I'm fine with either solution, but is flexibility in this case actually a benefit? Would you ever want an instance of the class to have a different notifier? This is one case where less flexibility may actually be better -- we want to know for sure what package the notice came from and reduce the possible ways to mess with that.

Reply:
You are correct, the notifier is not something we will encourage users to play with (though I'm not sure if there are any negative implications if they do?).

However, I had to make the `setNotifier` method public because of the way we export the js package (a singleton instance).
By making this method publicly accessible, we are prone to inconsistencies if we make the notifier as a static shared variable.
Consider this use case:
```
import hbClient from '@honeybadger-io/js'
hbClient.configure()

const anotherHbClient = hbClient.factory()
anotherHbClient.setNotifier({name: 'manual-notifier'})
```
With a static shared variable, both clients would have the same notifier name (manual-notifier).